### DISCUSSION TITLE
Create Windcalm fan with light is not showing status and timer not available

### DISCUSSION THREAD

**Discussion by WebfinityDesign** [make-all/tuya-local#1563] [Q&A]
I have the Create Windcalm fan with light. For the most part the integration is working perfectly. The only two things that don't work is the timer and the status: when I turn on the fan or the light with the remote or the Smartlife app the state in HA is still OFF or the other way around: turn of with the remote but the state in HA stays at ON.

Any solution for this?

**Comment by make-all** [comment]
The timer was simply not exposed (it is a read-only attribute on the fan, but you cannot set it).
I'm not sure what is causing the status not updating, maybe your log has some info.

**Comment by tobyt99** [comment]
@WebfinityDesign can you share your settings?  I  am having trouble getting the local connection to control speed.
